src/rtpstream.cpp

```diff
@@ -886,8 +886,10 @@ int rtpstream_get_videoport (rtpstream_callinfo_t *callinfo)
 }
 
 /* code checked */
-void rtpstream_set_remote (rtpstream_callinfo_t *callinfo, int ip_ver, char *ip_addr, int audio_port, int video_port)
+void rtpstream_set_remote (rtpstream_callinfo_t *callinfo, int ip_ver, char *ip_addr,
+                           int audio_port, int image_port, int video_port)
 {
+  // TODO: we don't do anything with image_port right now..
```

Comment:
Worth a note to say what we should do with it in future (and what the impact of not doing anything is)?